webui/vite.config.js

```diff
@@ -47,7 +44,10 @@ export default ({ command }) => {
             secure: false
           }
         }
-      }
+      },
+      build: {
+        sourcemap: 'inline',
```

Comment:
did you tried to turn it off to see if it remove the requirements for --max-old-space-size for the production build?

Reply:
@nopcoder The default is off. This is just for the dev build.
Again, even if it does help, it's temporary. I don't see the point in avoiding it. You're just kicking the can down the road. I'd rather handle it now, while I'm in the context than in two months when we'll have to have the entire discussion again 😉